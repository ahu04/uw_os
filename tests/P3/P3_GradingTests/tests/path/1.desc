Using execvp.  Score: 4