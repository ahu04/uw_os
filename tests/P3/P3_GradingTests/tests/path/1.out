${PROMPT}myprint is called
${PROMPT}